benefits/urls.py

```diff
@@ -16,12 +16,18 @@
 handler404 = "benefits.core.views.page_not_found"
 handler500 = "benefits.core.views.server_error"
 
+
+def trigger_error(request):
+    raise RuntimeError("Test error")
+
+
 urlpatterns = [
     path("", include("benefits.core.urls")),
     path("eligibility/", include("benefits.eligibility.urls")),
     path("enrollment/", include("benefits.enrollment.urls")),
     path("i18n/", include("django.conf.urls.i18n")),
     path("oauth/", include("benefits.oauth.urls")),
+    path("sentry-debug/", trigger_error),
```

Comment:
Open to opinions about whether this endpoint should always be available, or only in `DEBUG` mode, or…

Reply:
Yeah I think this should only be registered for `DEBUG` mode.